ui/jstests/test-static-assets.jsx

```diff
@@ -0,0 +1,159 @@
+define(['QUnit', 'jquery', 'static_assets', 'reactaddons',
+  'test_utils', 'jquery_mockjax'], function(
+  QUnit, $, StaticAssets, React, TestUtils) {
+    'use strict';
+
+    var StaticAssetsPanel = StaticAssets.StaticAssetsPanel;
+    var waitForAjax = TestUtils.waitForAjax;
+
+    var staticAssetsResponse = {
+      "count": 2,
+      "next": null,
+      "previous": null,
+      "results": [
+        {
+          "id": 1,
+          "asset": "http://lore.test.mit/media/assets/edX/toy/" +
+            "TT_2012_Fall/subs_CCxmtcICYNc.srt_dZF2PcC.sjson",
+          "name": "subs_CCxmtcICYNc.srt_dZF2PcC.sjson"
+        },
+        {
+          "id": 2,
+          "asset": "http://lore.test.mit/media/assets/edX/toy/" +
+            "TT_2012_Fall/test_YpcBO8Y.txt",
+          "name": "test_YpcBO8Y.txt"
+        }
+      ]
+    };
+
+    QUnit.module('Test static assets panel', {
+      beforeEach: function() {
+        TestUtils.setup();
+
+        TestUtils.initMockjax({
+          url: '/api/v1/repositories/repo/learning_resources/1/static_assets/',
+          type: 'GET',
+          responseText: staticAssetsResponse
+        });
+      },
+      afterEach: function() {
+        TestUtils.cleanup();
+      }
+    });
+
+    QUnit.test(
+      'Assert that StaticAssetsPanel loads properly',
+      function(assert) {
+        var done = assert.async();
+        var afterMount = function(component) {
+          // wait for calls to populate page
+          waitForAjax(1, function () {
+            // two static assets
+            var $node = $(React.findDOMNode(component));
+            var $vocabSelect = $node.find("li");
+            assert.equal($vocabSelect.size(), 2);
+
+            done();
+          });
+        };
+        React.addons.TestUtils.renderIntoDocument(<StaticAssetsPanel
+          repoSlug="repo"
+          learningResourceId="1"
+          ref={afterMount} />);
+      }
+    );
+
+    QUnit.test(
+      'Assert that StaticAssetsPanel list contains links',
```

Comment:
Can we merge this test with the previous one? It looks like there is just one extra assert